templates/music/includes/music_detail_stats.html

```diff
@@ -0,0 +1,107 @@
+{% load music %}
+
+<div class="tabs-panel" id="music-stats">
+
+    <h3>Estimated Revenue</h3>
+    <div class="profile-container">
+        <canvas id="myChart"></canvas>
+    </div>
+    <h3>Streams</h3>
+    <div class="profile-container">
+        <canvas id="myChart2"></canvas>
+    </div>
+    <div class="break-small">
+        <hr>
+    </div>
+    <h3>Downloads</h3>
+    <div class="profile-container">
+        <canvas id="myChart3"></canvas>
+    </div>
+    <div class="break-small">
+        <hr>
+    </div>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.1.4/Chart.min.js"></script>
```

Comment:
JS should actually probably remain in the original file instead of being moved over to this `include/` template. That way it can be in the `{% block extrajs %}` block.